_docs/_latest/setup/upgrade.md

```diff
@@ -401,6 +401,100 @@ Forseti is updated to be usable on a non organization resource.
 {% endcapture %}
 {% include site/zippy/item.html title="Upgrading 2.3.0 to 2.4.0" content=upgrading_2_3_0_to_2_4_0 uid=5 %}
 
+{% capture upgrading_2_4_0_to_2_5_0 %}
+
+Starting v2.5, Forseti Inventory will be integrated with the Cloud 
+Asset Inventory (CAI) service if the user is deploying Forseti. CAI
+feature is supported only if the level is `organization`.
+
+Users upgrading to v2.5 from v2.4 can enable CAI by following the steps below:
+1. Update the `deploy-forseti-server.yaml` file under `deployment-templates`
+with the name and location of the newly created bucket. Make sure the location 
+is same as the location of other forseti-bucket in `deployment-templates`.
+    ```
+    imports:
+    - path: storage/bucket_cai.py
+      name: bucket_cai.py
+  
+    resources:
+    
+    # Cloud Storage
+    - name: {FORSETI_BUCKET}
+      type: bucket.py
+      properties:	
+        location: {BUCKET_LOCATION}
+    - name: {FORSETI_CAI_BUCKET}
+      type: bucket_cai.py
+      properties:
+        location: {BUCKET_LOCATION}
+        retention_days: 14
+    ```
+1. Users can enable CAI if the level is `organization` by updating the 
+`forseti_conf_server.yaml` file with the location of the newly created
+bucket.
+   ```
+    cai:
+        enabled: true    
+        gcs_path: gs://{FORSETI_CAI_BUCKET}
+   ```
+1. Users need to update the quota by modifying the `forseti_conf_server.yaml`
+file as shown below.
+    ```
+    cloudasset:
+        max_calls: 1
+        period: 1.0
+    ```
+1. Run the below command on cloud shell. It assigns `roles/storage.objectAdmin` 
```

Comment:
There is a step missing. The forseti server service account also needs to add the roles/cloudasset.viewer role to their Organization IAM policy.